### DISCUSSION TITLE
Collaboration Design Patterns > Relations between Player Actions > Synergies

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#112] [Framework Discussions]
# Collaboration Design Patterns > Relations between Player Actions > Synergies

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
In this case, the act of setting a ship on fire, causing it to flood or damaging a specific module can help an ally in destroying it or to be able to flee from it
